File: fatorial.c
#include <stdio.h>
#include <stdlib.h>

int Fat(int n) {
    if (n == 0 || n == 1) {
        return 1;
    } else {
        return n * Fat(n - 1);
    }
}

int main() {
    int n = 0;
    printf("Digite um número para o cálculo de seu fatorial: \n");
    scanf("%d", &n);

    printf("O fatorial de %d é: %d\n", n, Fat(n));
    return 0;
}

File: Fatorial.py
def Fat(n):
    if n == 0 or n == 1:
        return 1
    else:
        return n * Fat(n - 1)

print("Digite um número para o cálculo de seu fatorial:")
a = int(input())
print(f"O fatorial de {a} é :{Fat(a)}")
